packages/core/src/Component.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+export interface Component {
```

Comment:
Placeholder for later impl, incrementally landing smaller changes. Needed for test/dependency checks to pass.